docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-google-cloud-functions.md

```diff
@@ -6,31 +6,24 @@ keywords: [how to, deploy a pipeline, Cloud Function]
 
 # Deploy a pipeline with Google Cloud Functions
 
-This guide shows you how to deploy a pipeline using the gcloud shell and `dlt` CLI commands. To deploy a pipeline using this method, you must have a working knowledge of GCP and its associated services, such as Cloud Functions, Cloud Source Repositories, Shell Editor, IAM and permissions, and GCP service accounts.
+This guide shows you how to deploy a pipeline using the gcloud shell and dlt CLI commands. To deploy a pipeline using this method, you must have a working knowledge of GCP and its associated services, such as Cloud Functions, IAM and permissions, and GCP service accounts.
```

Comment:
are we going with dlt over `dlt`?

Reply:
https://www.notion.so/dlthub/Documentation-Writing-Guide-87f4fcc32655460c83bcaf7787d11e67?pvs=4#1299fb8e23cf80488adfe5fdde27907b